contracts/mocks/StringsMock.sol

```diff
@@ -5,19 +5,19 @@ pragma solidity ^0.8.0;
 import "../utils/Strings.sol";
 
 contract StringsMock {
-    function fromUint256(uint256 value) public pure returns (string memory) {
+    function toStringDecimal(uint256 value) public pure returns (string memory) {
```

Comment:
why not name it "toString" ... just like the underlying library function ?

Reply:
Because when calling `await strings.toString(input)` JS gets confused and instead of calling the contract it just returns a string `"[Object object]"`.

Reply:
I'm fixing that